docs/best-practices.md

```diff
@@ -3,32 +3,8 @@ Best Practices
 
 This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
 
-
-## Custom Component
```

Comment:
Your call, while reading I was like "Wait you have best practices, I am building a custom component and I need to go on another page?".